app/models/katello/content_view_version.rb

```diff
@@ -109,6 +109,14 @@ def description
       history.publish.successful.first.try(:notes)
     end
 
+    def description=(new_description)
+      if description
```

Comment:
I thought that this code was only run after a successful publish, which creates a `history.publish.successful.first`, so it should never be nil (right?).